packages/terra-worklist-data-grid/src/proptypes/columnHeaderCellShape.js

```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types';
+
+const columnHeaderCellShape = PropTypes.shape({
+  /**
+   * String representing a unique identifier for the column header cell.
+   */
+  id: PropTypes.string.isRequired,
+  /**
+   * String of text to render within the column header cell.
+   */
+  displayName: PropTypes.string,
+  /**
+   * Boolean value indicating whether or not the cell can accept focus.
+   */
+  acceptsFocus: PropTypes.bool,
```

Comment:
Is there a reason that a column headers would not accept focus?

Reply:
In the event that headers provide no functionality, we do not see the need to give them focus. As we discussed, the determination to accept focus can be computed based on specific props. For example, a column that does not allow sorting or resizing need not accept focus. @chrismichalewicz & @eawww, please weigh in on this suggestion and include it for the story that deals with this functionality.